src/utils/localStorage.ts

```diff
@@ -0,0 +1,21 @@
+export const loadState = () => {
+	try {
+		const serializeState = localStorage.getItem('state');
+		if (serializeState === null) {
+			return undefined;
```

Comment:
maybe we use a empty {} instead undefined? i think it's may be a safer way for us

Reply:
and plz resolve a conflicts